Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -384,4 +356,269 @@ static void _GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameSi
     }
 
     return lines;
+}
+
+/**
+ * Helper method to retrieve font fmaly names installed in the system.
+ *
+ * @return Unmutable array of font family name strings that are installed in the system.
+ */
+static NSArray<NSString*>* _DWriteGetFontFamilyNames() {
+    NSMutableArray<NSString*>* fontFamilyNames = [NSMutableArray<NSString*> array];
+
```

Comment:
It was more that I _already_ have the CF versions of some of these functions in another branch, so I might as well move them over now. It won't be too much effort.

---
In reply to: [78617853](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78617853) [](ancestors = 78617853,78611079,78610729,78225402,78221279,78220974)